tests/conftest.py

```diff
@@ -127,3 +142,39 @@ def _message_maker(frozen_time, zoom_event):
         return msg
 
     return _message_maker
+
+
+@pytest.fixture(autouse=True)
```

Comment:
There did not seem to be a way to create a single fixture. `mock.patch` only allows one target when used this way. `autouse` is nice because the mock gets activated for all functions. If you need to check call counts, etc, you can still pull the fixture into the test function scope by including it in the arg list (like I did in some of the webhook tests).